src/models/view/right-nav.ts

```diff
@@ -44,7 +44,11 @@ export const NavTabSectionModel =
   .views(self => ({
     showStarsForUser(user: UserModelType) {
       return user.type && (self.showStars.indexOf(user.type) !== -1);
-    }
+    },
+    showDeleteForUser(user: UserModelType) {
+      // allow teachers to delete supports
+      return (user.type === "teacher") && (self.type === ENavTabSectionType.kTeacherSupports);
```

Comment:
My preference would be to handle this the way `showStarsForUser()` is handled, i.e. with a specification in `app-config.json`, both for consistency and because down the road someone's surely going to ask us to allow students to delete their own documents from the thumbnail views and it'd be nice to just turn it on in `app-config.json`.

Reply:
I was going down that path but I pulled up short namely because we would then need to sync properties for all documents which we don't do now (only the content is synced).  I didn't want to make that big of a change in case there are side effects.  I agree a generic config item like `"allowDelete": ["teacher"]` would be nice.

Reply:
👍 Then perhaps a comment in the code to that effect so that if/when we come back to this we'll remember how we got here and what the issues were.